src/components/EventCreator/EventForm.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import { Container, Typography, Box, Card, Switch, FormControlLabel } from "@mui/material";
+import NoteTemplateForm from "./NoteTemplateForm";
+import PollTemplateForm from "./PollTemplateForm";
+
+const EventForm = () => {
+  const [isNote, setIsNote] = useState<boolean>(true);
```

Comment:
wonder if instead of a boolean switch, it should be a tabbed interface, it would scale better when we wish to create more things, but it's fine for the scope of this PR.